newsfragments/76.feature.rst

```diff
@@ -0,0 +1,9 @@
+Mark ABI types with optional attributes.
```

Comment:
We should figure out a better way to structure this as it doesn't seem super meaningful as a code block. I wonder if we can figure out how to bullet for release notes? I have no experience with that.
 
<img width="729" alt="Screenshot 2024-05-31 at 10 10 11" src="https://github.com/ethereum/eth-typing/assets/3532824/7d39f4da-e879-44b4-ab55-927354de5a50">

Reply:
It can be fine for now... maybe we can put a colon after the first sentence and let it just be a text block? Might be nice to figure out how to bullet at some point though. This isn't blocking by any means.

Reply:
Yeah that looks less than ideal but maybe the bullets are possible, I'll take a brief look at it.

Reply:
<img width="744" alt="Screenshot 2024-06-03 at 10 50 56 AM" src="https://github.com/ethereum/eth-typing/assets/435903/d8b5c161-9e51-4293-b73a-52fd4be06127">

Just using a list with `*`s seemed to work fine. Thanks for the callout! 👍